.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install build
+    - name: Build package
+      run: python -m build
+    - name: Publish package
+      uses: pypa/gh-action-pypi-publish@27b31702a0e7fc50959f5ad993c78deac1bdfc29
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
@thraxil do we have the PyPI secret token/password in our GitHub organization? I don't think we do. 

If not, could you please add it?

Reply:
I haven't put the PyPI token in the organization because I've been creating a new token per-project/integration so they  have separate audit trails, can be revoked indepedently, etc. I'm going to update this repo with a `django-tiers` specific token. (separate tokens are the best we can do currently since PyPI doesn't yet have a notion of "organization"s).

Reply:
Thanks @thraxil, I don't have admin access to this repo, so please let me know if and when should I change the code to accomodate for the github-stored secret.